source/ui/win32/include/win32.ooc

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2015 - Simon Mika <simon@mika.se>
+ *
+ * This sofware is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+include windows
+
+LPSTR: cover from Char*
+LPMSG: cover from MSG_OOC*
+LPARAM: cover from Long*
+LRESULT: cover from Long*
+WPARAM: cover from UInt*
+HINSTANCE: cover from Void*
+HBRUSH: cover from Void*
+HWND: cover from Void*
+HMENU: cover from Void*
+LPVOID: cover from Void*
+HICON: cover from Void*
+HCURSOR: cover from Void*
+HBITMAP: cover from Void*
+HDC: cover from Void*
+HGDIOBJ: cover from Void*
+
+PAINTSTRUCT_OOC: cover from PAINTSTRUCT
+POINT_OOC: cover from POINT
+RECT_OOC: cover from RECT
+MSG_OOC: cover from MSG {
```

Comment:
If you want to keep the same name as the original struct, use `PAINTSTRUCT: extern cover`.

If you want to give them your own names, then give them proper ooc names: `PaintStruct: cover from PAINTSTRUCT`.